Makefile

```diff
@@ -264,9 +264,12 @@ check:
 check-chpldoc: chpldoc third-party-test-venv
 	@bash $(CHPL_MAKE_HOME)/util/test/checkChplDoc
 
-install: comprt
+install:
 	@bash $(CHPL_MAKE_HOME)/util/buildRelease/install.sh --stage=${DESTDIR}
 
+# install-chpldoc:
+# 	@bash $(CHPL_MAKE_HOME)/util/buildRelease/install.sh --stage=${DESTDIR} --chpldoc
```

Comment:
should this stay commented-out?

Reply:
oops! It's an artifact from an attempt to make a separate install target for `chpldoc` - the right solution turned out just to let install scoop up whatever was already built using the specific `make <target>`